server/server.go

```diff
@@ -45,6 +45,8 @@ type Config struct {
 
 	LogLevel logging.Level
 	Log      logging.Interface
+
+	PathPrefix string
```

Comment:
Can you add a `f.StringVar(...` to the `RegisterFlags` function so users can set this on the command line?

Reply:
Done!